src/rules/converters/file-name-casing.ts

```diff
@@ -0,0 +1,56 @@
+import { RuleConverter } from "../converter";
+
+const IGNORE_CASE_NOTICE = "ESLint (Unicorn plugin) does not support ignore as case";
+export const convertFileNameCasing: RuleConverter = tslintRule => {
+    return {
+        rules: [
+            {
+                ruleName: "unicorn/filename-case",
+                ...collectArguments(tslintRule.ruleArguments),
+            },
+        ],
+        plugins: ["unicorn"],
+    };
+};
+
+const collectArguments = (ruleArguments: any[]) => {
+    const notices: string[] = [];
+    const casesMap = new Map();
+    casesMap.set("camel-case", "camelCase");
+    casesMap.set("pascal-case", "pascalCase");
+    casesMap.set("kebab-case", "kebabCase");
+    casesMap.set("snake-case", "snakeCase");
+    const foundCases: { [k: string]: any } = {};
```

Comment:
```suggestion
    const foundCases: { [k: string]: string } = {};
```

...right? Once `casesMap` is converted to a `{ [i: string]: string }`, this should work more easily in the typings.